CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `CadenzaClient#selectObjects`
 - `CadenzaChangeExtentEvent`
 - Documentation for the "JSON Representation of Cadenza Object Data"
+- `selectableLayers` parameter to `CadenzaClient#selectObjects` and relevant documentation 
```

Comment:
The whole method is new in this release (see three lines above), i.e. it's not necessary to mention this option specifically in the changelog.